app/controllers/accounts_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
         @statuses        = filtered_status_page(params)
         @statuses        = cache_collection(@statuses, Status)
 
+        @rss_url = rss_url
```

Comment:
Align assignment with `@statuses` (remove newline above) and add newline before conditional block.